bgpd/bgpd.h

```diff
@@ -159,6 +159,9 @@ struct bgp_master {
 	/* How big should we set the socket buffer size */
 	uint32_t socket_buffer;
 
+	/* Should we do wait for fib install globally? */
+	bool wait_for_fib;
```

Comment:
bgp_master_init() should set a default, right?

Reply:
fixed